client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -332,7 +332,9 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( {
 
 	return (
 		<div className="credentials-form">
-			<h3>{ translate( 'Provide your SSH, SFTP or FTP server credentials' ) }</h3>
+			<h4 className="credentials-form__heading">
```

Comment:
Updated level heading to respect accessibility standards. Added a class name for styling.